pipeline-templates/populate-cache-steps.yml

```
@@ -0,0 +1,11 @@
+steps:
+  - task: CacheBeta@0
+    displayName: 'Cache NuGet packages'
+    inputs:
+      key: nuget | **/packages.lock.json
+      path: $(NUGET_PACKAGES)
+  - task: CacheBeta@0
+    displayName: 'Cache Library Manager libraries'
+    inputs:
+      key: src/Stryker.Core/Stryker.Core/Reporters/HtmlReporter/Files/mutation-testing-elements.js
```

I think the key should probably be similar to the nuget key, like so:

```suggestion
      key: libman | **/libman.json
```

---

Resolved in commit d348a45.